dbt_llm_tools/vector_store.py

```diff
@@ -149,16 +149,15 @@ def get_models(self, model_ids: list[str] = None) -> list[DbtModel]:
         Returns:
             list[DbtModel]: A list of dbt model objects retrieved from the vector store.
         """
-        models = []
-        raw_models = self.__collection.get(ids=model_ids)
-
-        for i in range(len(raw_models["ids"])):
-            models.append(
-                {
-                    "id": raw_models["ids"][i],
-                    "document": raw_models["documents"][i],
-                }
-            )
+        model_embeddings = self.__collection.get(ids=model_ids)
```

Comment:
What does `chroma_db.collection.get()` return if there are no documents with IDs that match your query?

Reply:
i think it returns an empty GetResult object
```
class GetResult(TypedDict):
    ids: List[ID]
    embeddings: Optional[List[Embedding]]
    documents: Optional[List[Document]]
    uris: Optional[URIs]
    data: Optional[Loadable]
    metadatas: Optional[List[Metadata]]
```

Reply:
How does `len(empty GetResult)` behave in this instance? Does the function still return an empty array?

Reply:
should i test it? 
the only call to this function in in `/client/pages/vector.py`:
```
with view_tab:
    ...    
    stored_models = vector_store.get_models()
    ...
```
and here we don't supply the parameter `ids`, in which case chroma_db.Collection.get() is called with no arguments, and it returns all the models